### PR TITLE
Make search results link open in new tab

### PR DESCRIPTION
**Pull request by kelynch**

### CODE REVIEW

File: app/views/catalog/_report_biased_results_form.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
     You can help us address this issue by reporting any biased or harmful results you encounter on Library search tools by using the form below. If you would prefer to report this anonomously, you may leave the name end email fields blank.
     <br/>
     <br/>
-    You are reporting bias in the following <%= link_to 'search results', form.context %>
+    You are reporting bias in the following <%= link_to 'search results', form.context, target: '_blank' %>
```
Comment (sandbergja) [inline app/views/catalog/_report_biased_results_form.html.erb:10]: Could we please also include an indication to screen reader users that this opens in a new tab?  Either in the link text itself, or we could add a little icon like in #3634?

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @kelynch !  Just one accessibility suggestion

File: app/views/catalog/_report_biased_results_form.html.erb (hunk 1 above)
Comment (kelynch) [reply app/views/catalog/_report_biased_results_form.html.erb:10]: Great idea!  I'll do this after this morning's demo.

**Review by sandbergja — APPROVED** [review]
Confirmed that VoiceOver now specifies that this link opens in a new tab.  Thanks, @kelynch !

**Pull request merged**